api/handlers_e2e_test.go

```diff
@@ -1304,7 +1304,7 @@ func TestLookupInnerLogs(t *testing.T) {
 			require.NotNil(t, response.LogData)
 			ld := *response.LogData
 			require.Equal(t, 1, len(ld))
-			require.Equal(t, appCall.Txn.ID().String(), ld[0].Txid)
+			// require.Equal(t, appCall.Txn.ID().String(), ld[0].Txid)
```

Comment:
@algochoi could you help us update these new tests? It looks like there was a discrepancy between master and develop since we no longer have the original app calls

Reply:
sure thing, will look into it

Reply:
PR here: https://github.com/Eric-Warehime/indexer/pull/1

I wasn't sure how the refactoring was being done on a macro scale so feel free to make any stylistic changes as well